docs/configuration-format.md

```diff
@@ -0,0 +1,157 @@
+# OpenTelemetry Collector Configuration Format
+
+The effective configuration that defines runtime behavior of the collector is calculated from 2 sources: the Default configuration which is hard-coded and the User-defined configuration which supplied by the user via various configuration sources. Default configuration defines reasonable defaults that most users will likely want to use. User-defined configuration can override the defaults and specify additional configuration. The effective configuration is calculated by merging Default and User-defined configurations (see merging rules in a separate section below).
```

Comment:
The effective configuration is calculated by [merging Default and User-defined configurations](#Merging-Rules)

Reply:
Done